Doc/library/wsgiref.rst

```diff
@@ -119,7 +119,8 @@ in type annotations.
    applications to set up dummy environments.  It should NOT be used by actual WSGI
    servers or applications, since the data is fake!
 
-   Example usage::
+   Example usage, where the *start_response* callable should follow the
+   :class:`.StartResponse` protocol::
```

Comment:
I don't get this note. The `start_response` callable is provided by the server from `wsgiref`. It *does* follow the protocol, or we have a bug.

Reply:
I wanted to say that the implementation should call `start_response` using 2 inputs. I'll try to come up with a better wording.